src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CFTTaskMapperTest.java

```diff
@@ -1422,11 +1422,27 @@ void should_convert_task_resource_to_map_object_with_updated_attribute_names() {
             objectMapper.convertValue(taskResource, new TypeReference<HashMap<String, Object>>() {});
         Map<String, Object> camundaTaskAttributes = cftTaskMapper.getTaskAttributes(taskResource);
 
-        assertEquals(dbTaskAttributes.size(), camundaTaskAttributes.size());
+        assertEquals(dbTaskAttributes.size() - 15, camundaTaskAttributes.size());
```

Comment:
Is there a way we can deduce this number?  Failing that maybe just have the 15 as a variable with a descriptive name.